compat/src/suspense.js

```diff
@@ -67,6 +67,10 @@ function detachedClone(vnode, detachedParent, parentDom) {
 
 function removeOriginal(vnode, detachedParent, originalParent) {
 	if (vnode && originalParent) {
+		if (typeof vnode.type == 'string') {
```

Comment:
For DOM nodes we need to revalidate the props so that we re-attach event-listeners which are deleted by `unmount` as we do `dom._listeners = undefined`